bioio_ome_zarr/writers/ome_zarr_writer.py

```diff
@@ -14,41 +14,42 @@
     resize,
 )
 
-DimTuple = Tuple[int, ...]
-PerLevelDimTuple = List[DimTuple]
-LevelwiseSpec = Union[DimTuple, PerLevelDimTuple]
+DimSeq = Sequence[int]
```

Comment:
I am slightly hesitant on this one (DimSeq) because numpy's documentation states that array shapes are tuples.  Maybe some of the other reviewers have stronger opinions but I don't feel the need to make this one a generic Sequence.  Did you find that you had to convert from Sequence to tuple on some of the internals?  I'll see as I read further...

Reply:
Noticing now that other parts of the bioio apis return Tuples:
```
    def shape(self) -> Tuple[int, ...]

...

    def resolution_levels(self) -> Tuple[int, ...]
```
Sigh.